tests/cupy_tests/test_cusolver.py

```diff
@@ -177,3 +177,56 @@ def test_syevjBatched(self):
 
         self.assertEqual(v.shape, a.shape)
         self.assertEqual(w.shape, a.shape[:-1])
+
+
+@testing.parameterize(*testing.product({
+    'dtype': ['float32', 'float64', 'complex64', 'complex128'],
+    'n': [10, 40, 160],
+    'nrhs': [None, 1, 10],
```

Comment:
Since the feature simply wraps cuSOLVER, the parameters are too large for (non-`@slow`) unittest.

```suggestion
    'n': [3],
    'nrhs': [None, 1, 4],
```